### PR TITLE
Minor tweaks to wording and small MD corrections

### PR DESCRIPTION
**Pull request by PlagueHO**
Hi @gaelcolas and @johlju - just some minor proofing changes and tweaks to wording to reduce some of the long sentences.

### CODE REVIEW

File: content/Community/ownership.en.md [hunk 1]
```diff
@@ -12,21 +12,20 @@ author: "gaelcolas"
...
 The main purpose is to handle better the increased community contributions, and remove hindrance to improvements by spreading the load on a Committee, and giving more privileges to the maintainers.
 
 ## Context
 
 Initially, when DSC was new and the DSC Resource kit relatively small, it was possible for the DSC Team at Microsoft to handle the maintenance and releases of the DSC Resources part of the DSC Resource Kit.
 
-But the community has outgrown the capacity of the DSC team, while the latter is taking on more work with the new Azure Guest Configuration.
+However, the community has outgrown the capacity of the DSC team, while the latter is taking on more work with the new Azure Guest Configuration.
 
 The DSC Resources from the Resource Kit that were hosted under the PowerShell Team GitHub organization did not allow for external (non-Microsoft) contributors to be members of the organization, or given extra permissions beside what `external collaborators` allowed.
```
Comment (johlju) [inline content/Community/ownership.en.md]:
```suggestion
The DSC Resources from the Resource Kit that were hosted under the PowerShell Team GitHub organization did not allow for external (non-Microsoft) contributors to be members of the organization, or given extra permissions beside what the permission `external collaborators` allowed.
```

File: content/Community/ownership.en.md [hunk 2]
```diff
@@ -12,21 +12,20 @@ author: "gaelcolas"
...
 
 ## Committee and Teams
 
-By using its own GitHub Organization outside of Microsoft's ownership, the DSC Community can be steered by a committee where its members&mdash;including people not employed by Microsoft&mdash;have full control over the GitHub Organization.
+By creating a new DSC Community GitHub organization outside of Microsoft's ownership, the DSC Community will be steered by the DSC Community committee where its members&mdash;including people not employed by Microsoft&mdash;have full control over the GitHub Organization.
 
-For now, [the committee](../committee) is composed of Katie, Michael, Daniel & Johann, long time contributors and driving force of the DSC Resource Kit. This is formalizing what happened for a couple of years with the DSC Resource Kit.
+For now, [the committee](../committee) is composed of Katie, Michael, Daniel & Johann, long time contributors and driving force of the DSC Resource Kit. This is formalizing what has already been happening for the last couple of years within the DSC Resource Kit community.
```
Comment (johlju) [inline content/Community/ownership.en.md]:
```suggestion
For now, [the committee](../committee) is composed of Katie, Michael, Daniel & Johan, long time contributors and driving force of the DSC Resource Kit. This is formalizing what has already been happening for the last couple of years within the DSC Resource Kit community.
```

**Comment by johlju** [comment]
I think we should really split lines around 72-80 chars in the markdown, it's easier to make review comments then, and the contributor to see where the requested changes are.

**Comment by PlagueHO** [comment]
@johlju - agreed. We should actually set this in the markdown linting file so that it highlights in VSCode.

**Comment by gaelcolas** [comment]
Yep, just do it ;)

**Comment by johlju** [comment]
Added in PR #22.

**Comment by PlagueHO** [comment]
@johlju - should be good to go.

File: content/Community/ownership.en.md (hunk 2 above)
Comment (PlagueHO) [reply content/Community/ownership.en.md]: Done! Misspelling your name! :cry:

File: content/Community/ownership.en.md (hunk 1 above)
Comment (PlagueHO) [reply content/Community/ownership.en.md]: Done.

**Comment by johlju** [comment]
I really miss Reviewable. :/ I think we should add it. I add an issue for that.

**Pull request merged**